R/pages.R

```diff
@@ -25,3 +25,31 @@ page404 <- function(page = NULL, message404 = NULL){
     return(page)
   }
 }
+
+#' Fix conflicts when some bookmark uses bootstrap
+#'
+#' @description This function dynamically removes bootstrap dependency when user opens specified bookmark.
+#' It should be inserted in head of bootrstrap page.
+#' @param bookmark Bookmark name on which bootstrap dependency should be suppressed.
+#'
+#' @export
+disable_bootstrap_on_bookmark <- function(bookmark) {
+  bootstrap_dependency <- htmltools::renderDependencies(list(shiny:::bootstrapLib()), srcType = "href")
+  tagList(
```

Comment:
Please add `@import shiny` to docs.

Reply:
Based on: http://r-pkgs.had.co.nz/namespace.html I think we should make some code cleaning:
- remove `shiny::` across all functions
- add `@import shiny` to our namespace
But this should be included in separate PR. 
WDYT?

Reply:
Ok, so please just create an issue on that.